crates/bevy_pbr/src/cluster/mod.rs

```diff
@@ -511,7 +512,8 @@ pub(crate) fn clusterable_object_order(
 /// Extracts clusters from the main world from the render world.
 pub fn extract_clusters(
     mut commands: Commands,
-    views: Extract<Query<(Entity, &Clusters, &Camera)>>,
+    views: Extract<Query<(&RenderEntity, &Clusters, &Camera)>>,
+    mapper: Extract<Query<&RenderEntity>>,
```

Comment:
Can you explain to me what `mapper` is for?

Reply:
AFAIK it's to remap components with `Entity` fields, changing them from the main world id to the render world id. This is needed since ids are no longer stable across worlds

Reply:
Yeah makes sense